mlir/lib/Dialect/MIOpen/utility/loweringUtils.cpp

```diff
@@ -22,5 +23,46 @@ std::tuple<Value, ArrayAttr> untransform(OpBuilder &b, Value transformed,
   }
   return {ret, b.getArrayAttr(transformList)};
 }
+
+static llvm::StringMap<int64_t> canonicalizeDims(ArrayRef<int64_t> dims,
```

Comment:
Should this be in a `namespace {}` instead of `static`ing it? I can't tell if there's a style preference on that.

Reply:
I'll move them to `namespace {}` after fixing unit tests.